starlette/responses.py

```diff
@@ -19,9 +19,9 @@
 try:
     import ujson as json
 
-    JSON_DUMPS_OPTIONS = {"ensure_ascii": False}
+    JSON_DUMPS_OPTIONS: typing.Dict[str, typing.Any] = {"ensure_ascii": False}
```

Comment:
Can we avoid this?

Reply:
We need to define the type. I can do this if you want:

```python
JSON_DUMPS_OPTIONS = {"ensure_ascii": False}  # type: typing.Dict[str, typing.Any]
```